collie/model/base/trainer.py

```diff
@@ -169,11 +173,17 @@ class CollieMinimalTrainer():
         How often to log within steps, if ``logger`` is enabled
     flush_logs_every_n_steps: int
         How often to flush logs to disk, if ``logger`` is enabled
+    max_depth: int
+        Maximum depth of modules to show, use -1 to show all modules or 0 to show no summary.
+        Related to PyTorch Lightning's ``Trainer.enable_model_summary`` argument.
     weights_summary: str
-        Prints a summary of the weights when training begins
+        Deprecated, replaced with ``max_depth``. Prints summary of the weights when training begins
+    detect_anomaly: bool
+        Enable anomaly detection for the autograd engine.
     terminate_on_nan: bool
-        If set to ``True``, will terminate training (by raising a ``ValueError``) at the end of each
-        training batch, if any of the parameters or the loss are NaN or +/- infinity
+        Deprecated. Replaced with ``detect_anomaly``. If set to ``True``, will terminate training
```

Comment:
To better match the docstring for `weights_summary` above this: 

```suggestion
        Deprecated, replaced with ``detect_anomaly``. If set to ``True``, will terminate training
```